javascript/javascript2/week1/SpiritNameGen.js

```diff
@@ -0,0 +1,34 @@
+const myButton = document.querySelector("button");
+let inputText;
+const spiritAnimalArray = [
+  "The Icy Bison",
+  "The Swift Armadillo",
+  "The Miniature Buffalo",
+  "The Capable Quail",
+  "The Gleaming Pig",
+  "The Enthusiastic Crane",
+  "The Amusing Penguin",
+  "The Sparkling Reindeer",
+  "The Expensive Caterpillar",
+  "The Intelligent Pony",
+];
+
+myButton.onclick = function (event) {
+  if (inputText.length > 0) {
+    const section = document.querySelector("section");
```

Comment:
in here if you declared your section, p outside of the function, than you would have access to it in both if and else statements and you wouldn't have to repeatedly search for the elements in the document.